app/controller.js

```diff
@@ -88,4 +88,19 @@ exports.patchArticle = (req,res,next) =>{
             next(err)
         }
     })
-}
\ No newline at end of file
+}
+
+exports.deleteComment = (req,res,next) =>{
+    const {comment_id} = req.params
+
+    deleteFromComments(comment_id).then(({rows})=>{
+        if(rows.length===0){next()}
```

Comment:
Generally any error handling should be taken care of in the model not the controller, and we wouldn't be invoking next straight up, we can throw a customer error which is picked up in the catch block down below which is where next is invoked. Our catch is at the end of the chain ready to pick up any errors thrown for any reason anywhere through this process.